spectroscopy/code_src/mast_functions.py

```diff
@@ -131,13 +131,21 @@ def JWST_get_spec_helper(sample_table, search_radius_arcsec, datadir, verbose,
             print("Nothing to download for source {}.".format(stab["label"]))
             continue
 
-        # Download (supress output)
+        # Download
+        # Observations.download_products prints an info message for every download, like:
+        #   "Downloading URL https://mast... to ./data/... ... [Done]"
+        # Suppress these by redirecting stdout to a trap.
```

Comment:
Just adding this comment for clarity. I know this because the HST call does the same thing.

Reply:
e6253e0bffa1b501fe7b4740341d8ec6ecfd773f